lib/product-import.coffee

```diff
@@ -282,49 +272,98 @@ class ProductImport
             skus.push(variant.sku)
     return _.uniq(skus,false)
 
-
   _isExistingEntry: (prodToProcess, existingProducts) ->
```

Comment:
This method is not used anymore, so you can remove it.

Reply:
removed here f51d5c4